### PR TITLE
Make errors visible to user

### PR DESCRIPTION
**Pull request by will-jac**
This PR makes any server errors that occur when submitting feedback visible to the user in the feedback modal.

### CODE REVIEW

File: src/components/Feedback/SendFeedback.ts [hunk 1]
```diff
@@ -52,23 +52,23 @@ export const sendFeedback = (rootState: RootState): Promise<FeedbackResponse> =>
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
       const responseJSON = JSON.parse(req.responseText);
 
-      if (req.status !== 200) {
+      let message: string = ''; 
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      if ('message' in responseJSON && typeof responseJSON.message !== 'string') {
```
Comment (navzam) [inline src/components/Feedback/SendFeedback.ts:57]: Should this be `===` instead of `!==`?

**Comment by will-jac** [comment]
@navzam should be resolved now!

File: express.js [hunk 2]
```diff
@@ -149,7 +149,16 @@ app.post('/feedback', (req, res) => {
...
 
-  const webhook = new WebhookClient({ url: hookURL });
+  let webhook;
+  try {
+    webhook = new WebhookClient({ url: hookURL });
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      message: 'An error occured on the server. If you are a developer, your webhook url is likely wrong.'
+    });
```
Comment (navzam) [inline express.js:159]: Needs a `return` after `res.status()` so that it doesn't continue on to the `webhook.send()`

**Review by navzam — COMMENTED** [review]
@will-jac sorry one more thing I didn't notice before

File: express.js (hunk 2 above)
Comment (will-jac) [reply express.js:159]: Should be resolved now! Sorry for the long delay

**Review by navzam — APPROVED** [review]

**Pull request merged**